README.md

```diff
@@ -119,6 +119,28 @@ var lightbox1 = $('.lighbox-1 a').simpleLightbox();
 var lightbox2 = $('.lighbox-2 a').simpleLightbox();
 ```
 
+### Customization
+
+You can customize Simplelightbox by changing the style in simplelightbox.css.
+
+If you are using SASS, you can customize Simplelightbox with the following variables
+
+```SASS
+$sl-font-family: inherit;

```

Comment:
I have changed `$sl-font-family` back to the original default, if you prefer this. However, I would recommend to leave the variable in for people who want to change the font family. I, for instance, thought of using an icon of Font Awesome or the likes. In the end, I would think that it is up to the programmer to test if the chosen font works across browsers and OSs.